syntax = "proto3";

message User {
  string user_id = 1; // Human-readable user ID
}
service UserServicer {
  // RPC methods go here
}
